webapp/src/store/tasks.ts

```diff
@@ -9,11 +9,16 @@ import { useToaster } from '@/composables/toaster'
 import { CONFIG } from '@/config'
 import { useTrainingStore } from './training'
 
+const TASKS_TO_FILTER_OUT = ['simple_face', 'cifar10']
```

Comment:
Dirty and ugly fix to hide some tasks in the webapp, is it worth implementing something better?

Reply:
it'll be nice to have theses at all, so we need to update tests at some point. but no rush there, let's keep that for the release.

Reply:
Nice to remove them or to keep them?